docs/specs/testvectors.md

```diff
@@ -21,6 +21,95 @@ This section contains test vectors. Each test vector in specifies the configurat
   084fed08b978af4d7d196a7446a86b58009e636b611db16211b65a9aadff29c5
   4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a
 
+## Test Vectors for Fiat-Shamir
+
+Let `p=115792089237316195423570985008687907853269984665640564039457584007908834671663` and `Fp` be the 4-word field defined by `p`.
+
+
+### Vector 1: WriteAll
+
+- Description: Using `Fp`, the test steps are to (a) initialize the transcript object with the 4-byte string `test`, (b) write an array of bytes of size 100 that contains the integers 0, 1, 2, ..., 99, write the field element '7', write the 2-long array of field elements `[8, 9]`, and
+finally (c) generate 16 field elements.  The field elements should be:
```

Comment:
There seem to be multiple undesignated steps between (b) and (c). I submit that all such steps should be lettered, that each designated step except the last (now designated `(c)`) should terminate in a semicolon, and that the last should terminate in a period. This comment applies to all similar blocks of steps.